docs/installation/prepare/microcontrollers/development-setup.md

```diff
@@ -0,0 +1,362 @@
+---
+title: "The Micro-RDK Development Setup"
+linkTitle: "Microcontroller Development Setup"
+weight: 50
+no_list: true
+type: docs
+image: "installation/thumbnails/esp32-espressif.png"
+imageAlt: "E S P 32 - espressif"
+images: ["/installation/thumbnails/esp32-espressif.png"]
+description: "Set up the Espressif ESP32 for development with the micro-RDK."
+# SMEs: Nicolas Menard
+---
+
+{{% readfile "/static/include/micro-rdk.md" %}}
+
+## Get Started
+
+Follow these steps to install and build the micro-rdk on your ESP32 for development:
+
+1. Install the [required software](#software-requirements)
+2. [Set up your development enviroment](#set-up-your-development-enviroment) manually or with Viam's Canon CLI utility _(recommended)_
+3. [Install the Micro-RDK](#install-the-micro-rdk)
+
+### Hardware Requirements
```

Comment:
There is now quite a bit of duplication - consider pulling this out into includes unless you're sure we won't forget to update it in both places